securedrop/journalist_app/admin.py

```diff
@@ -171,7 +172,7 @@ def edit_user(user_id):
 
             commit_account_changes(user)
 
-        password = make_password()
+        password = make_password(config)
```

Comment:
Thought: if during `__init__.create_app()` we replaced `app.config.from_object(config.SourceInterfaceFlaskConfig)` with `app.config.from_object(config)` then we could use the application instance `app.config['MY_VAR']` (or in a blueprint `current_app.config['MY_VAR']`) to access config instead of passing `config` around.

Reply:
thanks for the tip. I need to educate myself a bit more.